book/website/collaboration/new-community/new-recruitment.md

```diff
@@ -0,0 +1,524 @@
+
+
+
+
+## Advertising/dissemination
+
+Dissemination and getting your job under the eyes of as wide an audience as possible will be the key to attracting a diverse candidate pool. This can be done both internally and externally, in addition to processes the recruitment team at your organisation may already have in place. Key to this is targeted outreach to encourage under-represented groups to apply.
+
+
+### Leveraging internal expertise and networks
+
+Work with your Recruitment Team to facilitate connection to internal networks or interest groups across your organisation. For instance, involving the Equality, Diversity, and Inclusion (EDI) team and/or a staff networks relevant to disability, gender, LGBTQ+, race. Doing this early in the recruitment process demonstrates a genuine commitment to diversity and integration into the recruitment strategy from the start rather than being an afterthought. These internal experts will have valuable insights for identifying diverse talent pools, networks and job boards. They can help you provide targeted outreach to under-represented groups, give guidance on cultural competence and help you tailor your approach to attract a wider range of candidates.
+
+
+### Social media and outreach 
+
+Depending on your organisation’s social media policy, how you share the role online in your personal networks and brand accounts can also influence its reach. Sharing job postings in targeted LinkedIn groups (for example Diversity in Tech), and using targeted hashtags (for example #DiversityInTech, #WomenInAI) can give your advert a wider spread, reaching  candidates who may not otherwise encounter it.
+
+
+### Targeting external networks and organisations 
+
+
+Targeting external organisations and networks with a focus on underrepresented groups will broaden your reach. We have included some suggestions around this in the Resources section.
+
+
+## Checklist for recruitment 
+
+The checklist and timeline below outline a range of activities which can be undertaken to increase the diversity of your applicant pool. This list is not exhaustive so you will need to consult your organisation’s recruitment policies and recommendations. Having a specific timeline outlined and communicated from the start of the recruitment process is important not only for hiring teams but also for applicants. 
+
+
+### Before advertising 
+
+
+
+- [ ] Ensure effective communication  between hiring manager and recruitment team - know what your recruitment team is already doing (for example  where roles will be advertised) to avoid working in silos
```

Comment:
```suggestion
- [ ] Ensure effective communication  between hiring manager and recruitment team - know what your recruitment team is already doing (for example, where roles will be advertised) to avoid working in silos
```